### DISCUSSION TITLE
Description of counters

### DISCUSSION THREAD

**Discussion by GflowerG** [winsiderss/systeminformer#1078] [Questions / Answers] [answered]
**Is it possible to read the description of counters (columns) directly from the `Process Hacker` interface?** In the standard Windows Task Manager, the description is presented in a tooltip from the column header.

It's strange that I can read and **`copy`** the description of _Mitigation Policies_, but not for standard counters... I have to look for a counter in MMC or in WTM.

**Comment by dmex** [comment, marked as answer]
> Is it possible to read the description of counters (columns) directly from the Process Hacker interface?

It's possible but the amount of text required would be a bigger download than the whole project.
